sanic/response.py

```diff
@@ -4,8 +4,9 @@
 try:
     from ujson import dumps as json_dumps
 except:
-    from json import dumps as json_dumps
-
+    from json import dumps as jjson_dumps
```

Comment:
You should leave this as `json_dumps` since otherwise it will raise an exception every time in line 249 when using json instead of ujson.

Reply:
OK,I get your means